addons/website_event_track/static/src/js/service_worker.js

```diff
@@ -291,7 +306,7 @@ const processPendingRequests = async () => {
     let pendingRequest;
     while ((pendingRequest = pendingRequests.shift())) {
         const request = deserializeRequest(pendingRequest);
-        await fetch(request);
+        await fetchWithPrefetchHeader(request);
```

Comment:
This shouldn't be altered. `pendingRequests` (aka "syncable requests") refers to requests that will introduce a change on the server-side and then shouldn't be considered as "prefetch" requests.

Note: In this case, only the "toggle reminder" route is concerned.

PS: I agree, there is missing doc here...

Reply:
Thanks for the information, I don't know much about service workers. I have changed and it still works.

Notes: 
- I see that this function is called from an event listener "sync". No risk that the page stored in the cache be refreshed through this listener ? (then the page will be tracked while the user hasn't access the page)
- following a remark below, I have instead of X-Prefetch header, used X-Do-Not-Track header

Reply:
As mentioned above, I don't know much about service worker so I have a newbie question: 
Will the method processPendingRequests called by a listener "sync" not fetch URL not seen by the user (by refreshing the cache) ?
In that case I should add the X-Do-Not-Track header (not X-Prefetch anymore as explained above) to prevent the page being wrongly tracked.

Thanks for your advice

Reply:
Hello @pparidans, just to ping you about my question above. Thanks.

Reply:
Thanks for the explanation, I will left it without the header as suggested as it is concerning only one URL.